app/emoji/lib/rocketchat.js

```diff
@@ -1,5 +1,18 @@
+import { Meteor } from 'meteor/meteor';
+
 import { emojioneRender } from '../../emoji-emojione/lib/emojioneRender';
 
+let EmojiPicker;
+const removeFromRecent = (emoji) => {
+	if (!Meteor.isClient) {
+		return;
+	}
```

Comment:
Since `Meteor.isClient` doesn't change during runtime, I believe it's safe to do something like

```
const removeFromRecent = Meteor.isClient ? (emoji) => { ... } : () => {};
```